modules/running-insights-operator-gather-cli.adoc

```diff
@@ -35,11 +35,24 @@ spec:
 ----
 +
 --
-<1> Replace the `<your_data_gather>` with a unique name for your gather operation.
-<2> Enter individual gather operations to disable under the `gatherers` parameter. This example disables the `workloads` data gather operation and will run the remainder of the default operations. To run the complete list of default gather operations, leave the `spec` parameter empty. You can find the complete list of gather operations in the Insights Operator documentation.
+<1> Under *metadata*, replace `<your_data_gather>` with a unique name for the gather operation.
+<2> Under *gatherers*, specify any individual gather operations that you intend to disable. In the example provided, `workloads` is the only data gather operation that is disabled and all of the other default operations are set to run.
+When the `spec` parameter is empty, all of the default gather operations run.
 --
 
+[IMPORTANT]
+====
+Do not add a prefix of "periodic-gathering-" to the name of your gather operation because this string is reserved for other administrative operations and might impact the intended gather operation.
+====
+
 .Verification
 
 * Check that your new gather operation is prefixed with your chosen name under the list of pods in the `openshift-insights` project. Upon completion, the Insights Operator automatically uploads the data to Red Hat for processing.
 
+For more information about the available gather operations, see the "Additional resources" section.
+
+[role="_additional-resources"]
```

Comment:
My apologies. This should go into the Assembly (using-insights-operator.adoc). The section does not work at the module level because of the directory structure.

Reply:
Done. No worries! Thanks :-)